### PR TITLE
Add Relaunch64 v3.3.2

### PR DESCRIPTION
**Pull request by denpatin**
Relaunch64 is a powerful IDE for C64-assembler-coding.

### CODE REVIEW

File: Casks/relaunch.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'relaunch' do
```
Comment (vitorgalvao) [inline Casks/relaunch.rb]: Should be `relaunch64`, [with a comment](https://github.com/caskroom/homebrew-cask/blob/45a1bf6350d352172dd9876cfb36bae246652bb4/Casks/c3.rb#L2).

File: Casks/relaunch.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  version '3.3.2'
+  sha256 'e45c6c06caa20eefcb1d1c1f1a77135ab7a4eb183792eb6808d7f6c3de4fcca9'
+
+  url "http://cznic.dl.sourceforge.net/project/relaunch64/#{version}/Relaunch64_#{version}_OS_X_app.zip"
```
Comment (vitorgalvao) [inline Casks/relaunch.rb]: Needs [a comment](https://github.com/caskroom/homebrew-cask/blob/7f90417d48b89b91a713586f267f725cf9a72e01/Casks/universal-media-server.rb#L5).

Also, [the format is wrong](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md#sourceforge-urls).

**Comment by denpatin** [comment]
@vitorgalvao Thank you for help!

* Renamed the cask
* Added the comment about its name
* Changed the format for downloading from SF

**Comment by vitorgalvao** [comment]
Format is still wrong. Should be `http://downloads.sourceforge.net/sourceforge/relaunch64/Relaunch64_#{version}_OS_X_app.zip`.

**Comment by denpatin** [comment]
Yeah, sorry. Amended.

**Pull request merged**